lib/config/presets/internal/group.ts

```diff
@@ -444,6 +444,55 @@ const staticGroups = {
     ],
     ignoreDeps: [], // Hack to improve onboarding PR description
   },
+  remark: {
+    description: 'Group remark packages together.',
+    matchPackageNames: [
+      'remark-autolink-headings',
+      'remark-breaks',
+      'remark-comment-config',
+      'remark-contributors',
+      'remark-defsplit',
+      'remark-directive',
+      'remark-embed-images',
+      'remark-external-links',
+      'remark-footnotes',
+      'remark-frontmatter',
+      'remark-gemoji',
+      'remark-gfm',
+      'remark-git-contributors',
+      'remark-github',
+      'remark-heading-gap',
+      'remark-highlight.js',
+      'remark-html',
+      'remark-images',
+      'remark-inline-links',
+      'remark-language-server',
+      'remark-license',
+      'remark-lint-no-dead-urls',
+      'remark-lint',
+      'remark-man',
+      'remark-math',
+      'remark-message-control',
+      'remark-normalize-headings',
+      'remark-react',
+      'remark-reference-links',
+      'remark-rehype',
+      'remark-retext',
+      'remark-slug',
+      'remark-squeeze-paragraphs',
+      'remark-strip-badges',
+      'remark-textr',
+      'remark-toc',
+      'remark-unlink',
+      'remark-unwrap-images',
+      'remark-usage',
+      'remark-validate-links',
+      'remark-vdom',
+      'remark-yaml-config',
```

Comment:
```suggestion
```
These are already matched via `matchPackagePrefixes: ['remark-']`

Reply:
I know, but I wanted to call out the first party packages explicitly (similarly to FusionJS)

If this is not wanted, I can of course remove them

Reply:
can't we match those via source url? it would make the present a lot smaller and improves overall performance

Reply:
> I know, but I wanted to call out the first party packages explicitly (I thought I saw other configuration doing the same)

I did something similar when I created the `group:rubyOnRails` preset: [^1]

```json
{
  "packageRules": [
    {
      "groupName": "Ruby on Rails packages",
      "matchDatasources": [
        "rubygems"
      ],
      "matchPackageNames": [
        "actioncable",
        "actionmailbox",
        "actionmailer",
        "actionpack",
        "actiontext",
        "actionview",
        "activejob",
        "activemodel",
        "activerecord",
        "activestorage",
        "activesupport",
        "railties",
        "rails"
      ]
    }
  ]
}
```

[^1]: [Renovate docs, `group:rubyOnRails` preset](
https://docs.renovatebot.com/presets-group/#grouprubyonrails
)